.github/maintainers_guide.md

```diff
@@ -4,39 +4,53 @@ This document describes tools, tasks and workflow that one needs to be familiar
 this project. If you use this package within your own software as is but don't plan on modifying it, this guide is
 **not** for you.
 
-## Tools (optional)
+## Tools
 
-> Are there any build tools, dependencies, or other programs someone maintaining this project
-> needs to be familiar with?
+Across our scripts we use a few tools. Using `brew install`, you will probably need to install `gsed`, `fd`, and `ruby`.
 
-## Tasks
+You will also need [Hack](https://docs.hhvm.com/hack/getting-started/getting-started) installed to run `hh_*****` commands.
 
-### Testing
+We highly recommend using our editor of choice, VSCode, as it integrates well our included tasks. **Please install the VSCode Prettier extension to keep our codebase readable and prevent extra noise in commits.**
+
+Look in [/bin](/bin) for a variety of useful scripts.
 
-> How do you run the tests?
 
-### Generating Documentation (optional)
+## Tasks
+
+[VSCode tasks](https://code.visualstudio.com/docs/editor/tasks) are extremely useful when developing to quickly test changes in the grammar. We highly recommend using the ones we have included, which can be found [here](/.vscode/tasks.json). You can access them quickly using the command palette.
 
-> If the documentation is generated from source, how does someone run the generation?
-> Are the docs published on a website (GitHub Pages)?
+### Testing
+
+Use the `npm run` scripts to run our test suite. For example, `npm run test` or `npm run test-corpus`. 
 
 ### Releasing
 
-> A description of the process to make a release for this project. Do not share any secrets here.
+This project is released here: https://www.npmjs.com/package/tree-sitter-hacklang.
 
 ## Workflow
 
-### Versioning and Tags
+For making fixes or updates to the grammar, the process is generally as follows:
+
+1. Find an error (via reported issues, `bin/test-dir`, `bin/test-examples`, etc.).
```

Comment:
The numbering is off here

Reply:
Since it's a markdown list I just do `1.`s to it auto-increments

Reply:
Ahh github doesn't display it correctly on PRs. Looks good then!